src/stdatamodels/jwst/datamodels/wcs_ref_models.py

```diff
@@ -1057,10 +1222,23 @@ def validate(self):
 
 
 class WaveCorrModel(ReferenceFileModel):
+    """Model for the reference file for the wavecorr step."""
+
     reftype = "wavecorr"
     schema_url = "http://stsci.edu/schemas/jwst_datamodel/wavecorr.schema"
 
     def __init__(self, init=None, apertures=None, **kwargs):
+        """
+        Initialize the model.
+
+        Parameters
+        ----------
+        init : str, tuple, `~astropy.io.fits.HDUList`, ndarray, dict, None, optional
+            The data from which to initialize the model. Can be of any type that
+            is supported by DataModel, by default None
+        apertures : TODO, optional
+            TODO, by default None
```

Comment:
each aperture is a dict with keys 
"aperture_name" - str, the name of the aperture
"width" - float - aperture or pitch width in meters
"zero_point_offset" - astropy.Tabular2D model - a lookup table for the wavelength correction
"variance" - the variance of the correction